### PR TITLE
Update configure-windows-service-accounts-and-permissions.md

### PR DESCRIPTION
**Pull request by jpomfret**
Service SID for SQL Engine named instances was incorrect

(Tried to find this in the forked copy of the repo on my machine to make sure there were no other versions to update but it beat me today - hope this works ok)

### CODE REVIEW

**Comment by PRMerger16** [comment]
@jpomfret : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/database-engine/configure-windows/configure-windows-service-accounts-and-permissions.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ The following table shows permissions that [!INCLUDE[ssNoVersion](../../includes
...
+|**[!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]:**<br /><br /> (All rights are granted to the per-service SID. Default instance: **NT SERVICE\MSSQLSERVER**. Named instance: **NT Service\MSSQL$**_InstanceName_.)|**Log on as a service** (SeServiceLogonRight)<br /><br /> **Replace a process-level token** (SeAssignPrimaryTokenPrivilege)<br /><br /> **Bypass traverse checking** (SeChangeNotifyPrivilege)<br /><br /> **Adjust memory quotas for a process** (SeIncreaseQuotaPrivilege)<br /><br /> Permission to start SQL Writer<br /><br /> Permission to read the Event Log service<br /><br /> Permission to read the Remote Procedure Call service|
```
Comment (markingmyname) [inline docs/database-engine/configure-windows/configure-windows-service-accounts-and-permissions.md:267]:
```suggestion
|**[!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]:**<br /><br /> (All rights are granted to the per-service SID. Default instance: **NT SERVICE\MSSQLSERVER**. Named instance: **NT Service\MSSQLServer$**_InstanceName_.)|**Log on as a service** (SeServiceLogonRight)<br /><br /> **Replace a process-level token** (SeAssignPrimaryTokenPrivilege)<br /><br /> **Bypass traverse checking** (SeChangeNotifyPrivilege)<br /><br /> **Adjust memory quotas for a process** (SeIncreaseQuotaPrivilege)<br /><br /> Permission to start SQL Writer<br /><br /> Permission to read the Event Log service<br /><br /> Permission to read the Remote Procedure Call service|

```

**Comment by PRMerger7** [comment]
@markingmyname : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by markingmyname** [comment]
#sign-off

**Pull request merged**